about.html

```diff
@@ -0,0 +1,120 @@
+                <ul class="collapse-list">
+                    <li>
+                        <img id="close-btn" class="close-sign" src="images/close.png" alt="close sign">
+                    </li>
+                    <li>
+                        <a href="index.html" class="active-tab" target="_blank" rel="noopener">Home</a>
+                    </li>
+                    <li>
+                        <a href="#" id="about-link">About</a>
+                    </li>
+                    <li>
+                        <a href="index.html#main-program" target="_blank" rel="noopener">Program</a>
+                    </li>
+                </ul>
+            </nav>
+            <nav class="desktop-navbar">
+                <a href="index.html">
+                    <img class="desktop-logo" src="images/microverselogo.png" alt="microverse logo">
+                </a>
+                    <ul>
+                        <li>
+                            <a href="index.html">Home</a>
+                        </li>
+                        <li>
+                            <a href="about.html" class="active-tab">About</a>
+                        </li>
+                        <li>
+                            <a href="index.html#main-program">Program</a>
+                        </li>
+                        <li>
+                            <a href="https://apply.microverse.org/" target="_blank" rel="noopener">Join</a>
+                        </li>
+                    </ul>
+            </nav>
+        </header>
+
+        <section class="introduction">
+            <h1 class="introduction__header">
+                Microverse Full-Stack web development program
+            </h1>
+            <div class="introduction__description-container">
+                <p class="introduction__description">
+                    At Microverse, students learn through a very unique approach by pair programming and collaborating with other students in real-time, just as they would with colleagues of a distributed team in a real company.
+                    Students learn multiple programming languages and Computer Science fundamentals, through working on real projects with other students around the world.
+                </p>
+            </div>
+            <p class="contact-us-email">
+                Please contact us per Email for any further questions about Microverse program
+                <a class="contact-us-link" href="mailto:admissions@microverse.org">admissions@microverse.org</a>
+            </p>
+        </section>
+
+        <section class="alumni-opinion">
+            <h3>
+                What do alumni think of Microverse?
+            </h3>
+            <p class="alumni-description">
+                Microverse accepted its first cohort in early 2018 and its first graduates started receiving job offers in September 2018. Here are some of their comments, reviews, and videos. 
+            </p>
+
```

Comment:
- [ ] These elements are missing, please add them to comply with the project requirements:

![image](https://user-images.githubusercontent.com/38522578/137461410-444ea91b-8841-4940-a223-9758f5b4d7a1.png)